src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -54,6 +54,17 @@ internal static partial class ITypeSymbolExtensions
             return SyntaxFactory.RefType(refKeyword, underlyingType);
         }
 
+        public static TypeSyntax GenerateRefReadOnlyTypeSyntax(
+            this INamespaceOrTypeSymbol symbol)
+        {
+            var underlyingType = GenerateTypeSyntax(symbol)
+                .WithPrependedLeadingTrivia(SyntaxFactory.ElasticMarker)
```

Comment:
this elastic marker seems unnecessary.  You'll already get one for the refkeyword token.

Reply:
Not here, since this is following the same pattern as `GenerateRefTypeSyntax()`. Am I missing something?

---
In reply to: [141181425](https://github.com/dotnet/roslyn/pull/22285#discussion_r141181425) [](ancestors = 141181425)